### ISSUE TITLE
Allow execution of custom CloudFormation stacks

### ISSUE THREAD

**Issue by tj** [apex/up#218]
The tooling will be there, so might as well allow people to point to their own cloudformation files if they're limited by Up's.

**Comment by mrmark** [comment]
Would this include being able to supplement the current cloudformation used by Up?  For example, I like everything that Up makes, but would like to add DB, IAM roles, etc.

**Comment by tj** [comment]
@mrmark yep! I haven't thought much about this yet, but that would definitely be possible, and since stacks have nice input/output params it should be easy to combine them.

**Comment by mojodna** [comment]
IAM roles would be most helpful (for additional permissions beyond what's granted to the default role), either externally managed or as part of Up.
